tests_output/CHROME_86.0.4240.193_Windows_smoke_tests.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This also looks like generated output so best not to commit it. Possibly add:
```
/tests_output/
```
to `.gitignore`. This will ignore the whole directory.

Reply:
All changes done - thank you for the feedback .

Reply:
Hi Helen, I can see that you've added the appropriate entries to the `.gitignore` file but ideally you would now also delete those files that are already in the repository that we're now trying to ignore. Git won't delete them by default if you add them to the `.gitignore` in fact it continues to track them and allow you to commit changes to them as they are already part of the repo.

Super ideally we would re-write the history so that they were never included but they are only small files so it isn't too big a deal if they remain in the history.